cpp/msbuild/zeroc.datastorm.v140.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>zeroc.datastorm.v140</id>
     <title>ZeroC DataStorm C++ for Visual Studio 2015</title>
-    <version>1.0.0</version>
+    <version>1.1.0</version>
```

Comment:
Since it's a new version (1.1), we should drop support for older compilers.

Reply:
Fixed.